### PR TITLE
tests for password controller

### PR DESCRIPTION
**Pull request by gterzian**
tests for the password controller, also to provide a solid basis for https://github.com/stellar/stellar-client/issues/1178

### CODE REVIEW

**Comment by gterzian** [comment]
@nullstyle @bartekn   Ok good to go on my side

File: test/unit/specs/controllers/password-controller-test.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    scope.checkPassword();
+    expect(scope.status.passwordValid).to.be.true;
+    expect(scope.status.passwordConfirmValid).to.be.false;
+    expect(scope.errors.passwordErrors).to.be.empty;
+    expect(scope.errors.passwordConfirmErrors).to.be.empty;
+  });
+  
+  it('If there is no password, validateInput should mark that as an error', function () {
+    scope.data = {password: ''}
+    var validInput = scope.validators.pop()();
```
Comment (bartekn) [inline test/unit/specs/controllers/password-controller-test.js]: You should check all validators here. Right now there is only one but if more will be added this test may break.

**Comment by gterzian** [comment]
@bartekn allright thanks, how does this look? I think it should make the tests more resilient to changes, do you agree?

**Comment by bartekn** [comment]
Looks great now! Thanks for your contribution! :rocket:

**Pull request merged**